docs/issuer-node/issuer-node-guide.md

```diff
@@ -149,6 +149,14 @@ If you clicked on the **Issue credential** button, you will later be prompted to
 <div align="center">
 <img src= "/img/issue-credential-6.png" align="center" />
 </div>
+ 
+<br/>
+
+:::info
+There are 3 QR code options available: Link, Raw JSON and Base64 encoded. The preferred option is link because it is the most compact and the easiest to scan. It contains a link to download the original raw QR code content. Raw and Base64 encoded formats are useful for wallets that do not support Link mode or prefer not to follow that indirection.  
```

Comment:
> or **for privacy-cautious users**, who prefer not to follow that ~in~ **re**direction.

Reply:
But to receive a credential such users would anyway authenticate and expose their IP addresses.

Reply:
Good comment. Updated the info with this.